runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -94,6 +94,18 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
    bool isValueTypeClass(TR_OpaqueClassBlock *);
    bool isValueTypeClassFlattened(TR_OpaqueClassBlock *clazz);
    bool isValueBasedOrValueTypeClass(TR_OpaqueClassBlock *);
+   /** \brief
+    *	    Checks whether a class implements `IndentityObject`/`IdentityInterface`
+    *
+    *  \param clazz
+    *     The class that is to be checked
+    *
+    *  \return
+    *    `true` if the class implements `IndentityObject`/`IdentityInterface`;
```

Comment:
One minor nit: could you fix the spelling of "Identity" in this block please?   I won't re-run CI testing afterwards as it already passed and is not necessary for a comment change.

Reply:
Fixed in 3d52828